src/app/dashboard/page.tsx

```diff
@@ -337,9 +337,31 @@ const Home: NextPage = () => {
             </Box>
           </Flex>
         </GridItem>
+        <GridItem colSpan={{ base: 12, lg: 12 }}>
+          <Box mb={6}>
+            <GraphComponent
+              getRootProps={getHolderRootProps}
+              areaGraph={false}
+              graphData={transformHiIQHolderData(holderData)}
+              graphCurrentValue={numberOfIQHolder}
+              graphTitle="IQ Token Holders over time"
+              height={120}
+              isHolderGraph={true}
```

Comment:
You don't have to pass ``true`` to ``isHolderGraph``

Reply:
thanks, will change.